tests/src/Interop/NativeCallable/NativeCallableTest.csproj

```diff
@@ -37,5 +37,9 @@
   <ItemGroup>
     <Compile Include="NativeCallableTest.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- This is needed to make sure native binary gets installed in the right location -->
+    <ProjectReference Include="CMakeLists.txt" />
```

Comment:
@ramarag @sergiy-k @adityamandaleeka I heard from Sergiy that there is some magic being done to the scripts to copy the binaries and Rama/Aditya might know. Otherwise, I can dig a deeper into this.